doc/source/security/data-at-rest-encryption.rst

```diff
@@ -73,6 +73,8 @@ To install the selected plugin, follow the `installing and uninstalling plugins
 
     `KV Secrets Engine - Version 1 <https://www.vaultproject.io/docs/secrets/kv/kv-v1.html>`_
 
+    `KV Secrets Engine - Version 2 <https://www.vaultproject.io/docs/secrets/kv/kv-v2>`_
```

Comment:
Not a real issue, but just caught my perfectionist's eye.
```
https://www.vaultproject.io/docs/secrets/kv/kv-v1.html
https://www.vaultproject.io/docs/secrets/kv/kv-v2
```
I would prefer
```
https://www.vaultproject.io/docs/secrets/kv/kv-v1
https://www.vaultproject.io/docs/secrets/kv/kv-v2
```

Reply:
Fixed.